.php_cs

```diff
@@ -11,7 +11,12 @@ return PhpCsFixer\Config::create()
     ->setRules([
         '@Symfony' => true,
         '@Symfony:risky' => true,
+        '@PHP71Migration:risky' => true,
+        '@PHPUnit75Migration:risky' => true,
         'array_syntax' => ['syntax' => 'short'],
+        'ordered_imports' => true,
+        'declare_strict_types' => false,
+        'native_function_invocation' => true,
```

Comment:
Do we want to namespace them all, or only the ones for which it makes a difference for performance (which is the config included in `@Symfony:risky` ? The extra `\` in front of all calls adds visual noise. That's the reason for the rule selected by Symfony.

Reply:
I'm not a big fan of micro-optimization.
I think it's better to keep it consistent (adding front slash to everything or to nothing).
If we start to make a distinction, at every change we'll be forced to check whether the slash is worth or not. That's a burden.